heliopy/util/config.py

```diff
@@ -22,14 +23,14 @@ def get_config_file():
     config_filename = 'heliopyrc'
 
     # Get user configuration location
-    home_dir = os.path.expanduser("~")
-    config_file_1 = os.path.join(home_dir, '.heliopy', config_filename)
+    home_dir = pathlib.Path.home()
+    config_file_1 = home_dir / '.heliopy' / config_filename
 
-    module_dir = os.path.dirname(heliopy.__file__)
-    config_file_2 = os.path.join(module_dir, config_filename)
+    module_dir = pathlib.Path(heliopy.__file__).parent
+    config_file_2 = module_dir / config_filename
 
     for f in [config_file_1, config_file_2]:
-        if os.path.isfile(f):
+        if pathlib.Path.is_file(f):
             return f
```

Comment:
To fix everything elsewhere, we could just convert `f` back to a string here for now?